application/base_script.sh

```diff
@@ -0,0 +1,40 @@
+xcp_d \
+    --participant_label 01
+    -t rest \
+    --bids-filter-file filter.json \
```

Comment:
What is being filtered in this file?

Reply:
Potentially nothing. We can include the filter to only process some tasks 9e.g., just rest), if folks think that would be useful. It would at least exhibit the filter file functionality.

Reply:
That's fair. I'll comment it out.